community/kernel/src/docs/dev/batchinsert.asciidoc

```diff
@@ -57,3 +76,12 @@ component=neo4j-kernel
 source=examples/BatchInsertDocTest.java
 tag=configFileInsert
 ----
+
+=== Importing into an existing database ===
+
+Although it's a less common use case, the batch inserter can also be used to import data into an existing database.
```

Comment:
@tinwelint 

The line explains that you need to shutdown the existing database first:

> However, you will need to ensure that the existing database isn't in use while you write to it.

Do you think we need to make that wording more explicit?